schema.sql

```diff
@@ -1,11 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id integer,
+    id integer primary key generated always as identity,
     name varchar(100),
     date_of_birth date,
     escape_attempts integer,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species varchar(40)
```

Comment:
- [ ] Kindly add a new column called species with a string type using Alter Query. You should not edit the existing table instead create a new query.